tmeister/static/src/toggles/envs/select-envs.component.js

```diff
@@ -23,13 +48,9 @@ function toPillboxItem(env) {
   };
 }
 
-function setSelectedEnvs(action, envItems) {
-  const envs = chain(envItems)
-    .map(item => item.label)
-    .concat("production")
-    .uniq()
-    .value();
-
-  window.localStorage.setItem(`selected-envs`, JSON.stringify(envs));
-  action(envs);
+function useInitializeSelectedEnvs(setEnvs) {
+  useEffect(() => {
+    setEnvs(getSelectedEnvs())
+  }, [])
```

Comment:
It doesn't matter because setEnvs is a function created by `useState` so it's guaranteed to never change.

I could change it but I like it being empty because I intentionally _want_ this to only run once.